android/app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -3,19 +3,10 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item
-        android:id="@+id/mapsFragment"
+        android:id="@+id/findMealFragment"
         android:icon="@drawable/ic_baseline_map_24"
         android:title="@string/map_tab_label" />
 
-    <item
-        android:id="@+id/addMealFragment"
-        android:icon="@drawable/ic_baseline_add_24"
-        android:title="@string/add_meal_tab_label" />
-
-    <item android:id="@+id/findMealFragment"
-        android:icon="@drawable/ic_baseline_list_24"
-        android:title="@string/title_switch_to_list"/>
-
     <item
         android:id="@+id/settingsFragment"
         android:icon="@drawable/ic_baseline_settings_24"
```

Comment:
This is going to conflict with Fidel's PR with the refactoring of the FindMealFragment, no?

Reply:
Pretty sure that's going to be conflicted.